tests/double_key_tests.rs

```diff
@@ -4,9 +4,9 @@
 //
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
-use dusk_bls12_381::BlsScalar;
 use dusk_pki::SecretKey;
-use schnorr::{Proof, PublicKeyPair};
+use dusk_plonk::bls12_381::BlsScalar;
```

Comment:
Same here and in the rest of the file. But before proceed let's discuss in the chat, I think we all agreed; but let's double check.